azure-pipelines.yml

```diff
@@ -1,18 +1,18 @@
-# Scope the validations to master and PRs to master
+# Scope the validations to main and PRs to main
 trigger:
-  - master
+    - main
```

Comment:
any reason for the extra spaces?

Reply:
an autoformatter when saving 😠
Should be fixed now